update-git-submodules/src/index.ts

```
@@ -85,6 +87,31 @@ export function buildPrBody(
+		if (!match) continue;
+		const [, marker, path] = match;
+		if (marker === "-") uninitialized.push(path);
+	}
+	if (uninitialized.length === 0) return;
+	throw new Error(
+		`Submodule path(s) not initialized: ${uninitialized.join(", ")}. ` +
+			"Either set `init: true` on update-git-submodules (with a `token` " +
+			"that can clone the submodule repos), or set `submodules: recursive` " +
+			"on actions/checkout before this step.",
+	);
+}
```

ESLint

[error] Expected "assertSubmodulesInitialized" to come before "buildPrBody".

(perfectionist/sort-modules)